abi/actor.go

```diff
@@ -12,6 +16,18 @@ func (e ActorID) String() string {
 	return strconv.FormatInt(int64(e), 10)
 }
 
+// ProverID returns a 32 byte proverID used when computing ReplicaID
+func (e ActorID) ProverID() ([32]byte, error) {
```

Comment:
Instead of defining this on the `ActorID` type, can we make it a `ProverIDFromMinerIDForPoRepReplicaID` (or some less ridiculous name) helper that we put in `sector.go` or something?

Reply:
Why? That method would be taking the ActorID anyways, except now would be harder to discover, and have a longer name.

Reply:
Because `ActorID` doesn't have anything to do with PoRep / Replicas / proofs. It's not miner-specific, just the key of the state tree, so defining some method on it for the sake of unrelated code that 99% of its users won't actually care about seems bad.